src/main/java/io/github/moulberry/notenoughupdates/commands/dev/EnableStorageCommand.java

```diff
@@ -34,6 +37,7 @@ public EnableStorageCommand() {
 	public void processCommand(ICommandSender sender, String[] args) throws CommandException {
 		NotEnoughUpdates.INSTANCE.config.storageGUI.enableStorageGUI3 = true;
 		NotEnoughUpdates.INSTANCE.saveConfig();
+		Minecraft.getMinecraft().thePlayer.addChatMessage(new ChatComponentText(EnumChatFormatting.YELLOW + "[NEU] Enabled custom storage GUI"));
```

Comment:
dont we have a util for this

Reply:
yes but nea will change it anyways in another pr

Reply:
So why not just use the current utils method since refactoring is easier when we have no `Minecraft.getMinecraft().thePlayer.addChatMessage` stuff anywhere?

Reply:
:skull: